File: Programacion/UD3/ejercicios1 tema3/src/ejercicio1.java
import java.math.*;
import java.util.*;

public class ejercicio1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Dime cuantas pesetas tienes");
        if (sc.hasNextDouble()) {
            double pesetas = sc.nextDouble();
            sc.nextLine();
            if (pesetas <=0 ) {
                System.out.println("Error, el numero tiene que ser mayor que 0");
            } else {
                System.out.println( pesetas + " pesetas son " + String.format("%.2f",pesetas/166.386) + " euros");
            }

        } else {

            System.out.println("Error, abortando");
        }
    }
}

File: Programacion/UD7/EjerciciosKotlin1/src/Ejercicio1.kt
import com.sun.tools.javac.Main
import java.util.Scanner

fun main() {

    var sc = Scanner(System.`in`)

    println("Dime el primer numero")
    var num1: Int = sc.nextInt()
    println("Dime el segundo numero")
    var num2: Int = sc.nextInt()

    if (num1 == num2){
        println("Son iguales")
    } else {
        if (num1 > num2){
            println("$num1 es mayor que $num2")
        } else {
            println("$num2 es mayor que $num1")
        }
    }


}
